pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, obs-studio
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
The `finalAttrs`-pattern is currently preferred over `rec` when using `stdenv.mkDerivation`.
See https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes

```patch
diff --git a/pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix b/pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix
index 33e0ee2cbef7..350c9b8ad088 100644
--- a/pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix
+++ b/pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix
@@ -6,13 +6,14 @@
 , qtbase
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "obs-freeze-filter";
   version = "0.3.3";
+
   src = fetchFromGitHub {
     owner = "exeldro";
     repo = "obs-freeze-filter";
-    rev = version;
+    rev = finalAttrs.version;
     sha256 = "sha256-CaHBTfdk8VFjmiclG61elj35glQafgz5B4ENo+7J35o=";
     fetchSubmodules = true;
   };
@@ -39,4 +40,4 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
     maintainers = with maintainers; [ pschmitt ];
   };
-}
+})
```

Reply:
Done: 1f0e004f2b23f32568370b84046dd99fc1c6c5f4